integrated_channels/canvas/client.py

```diff
@@ -12,8 +14,21 @@ class CanvasAPIClient(IntegratedChannelApiClient):
     """
     Client for connecting to Canvas.
 
-    Code to obtain access tokens and posting to the courses and
-    completion status endpoints.
+    Required Canvas auth credentials to instantiate a new client object.
+        -  canvas_base_url : the base url of the user's Canvas instance.
+        -  client_id : the ID associated with a Canvas developer key.
+        -  client_secret : the secret key associated with a Canvas developer key.
+        -  refresh_token : the refresh token token retrieved by the `oauth/complete`
+        endpoint after the user authorizes the use of their Canvas account.
+
+    Order of Operations:
+        Before the client can connect with an Enterprise user's Canvas account, the user will need to
+        follow these steps
+            - Provide the ECS team with their developer key's client ID and secret.
+            - ECS will return a url for the user to visit which will prompt authorization and redirect when
+            the user hits the `confirm` button.
+            - The redirect will hit the `oauth/complete` endpoint which will use the provided oauth code
+            to request the access token necessary to post/delete/update courses and save it to the model.
```

Comment:
I think there is another step here which is to get the refresh token and save it to the model?

Reply:
Haha whoops, I got mixed up and left out the step 🙃